internal/olm/operator/manager.go

```diff
@@ -72,6 +74,10 @@ func (c *OLMCmd) newManager() (*operatorManager, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to get namespace from kubeconfig %s: %w", c.KubeconfigPath, err)
 	}
+	if m.olmNamespace = c.OLMNamespace; m.olmNamespace == "" {
+		m.olmNamespace = olm.DefaultNamespace
+	}
+
```

Comment:
Why it is done in this way? Shoud not it be:

```
	if len(m.olmNamespace) < 0 {
		m.olmNamespace = olm.DefaultNamespace
	}
```

Reply:
this should work as well. But I was following existing code https://github.com/operator-framework/operator-sdk/blob/master/internal/olm/operator/manager.go#L78 to make the code look alike